modules/cxf/src/test/resources/META-INF/mule-module.properties

```diff
@@ -9,6 +9,14 @@ artifact.export.classPackages=org.mule.runtime.module.cxf.example,\
                               org.mule.runtime.module.cxf.employee,\
                               org.mule.runtime.module.cxf.jaxws,\
                               org.mule.extension.http.api,\
-                              org.mule.extension.http.api.server
+                              org.mule.extension.http.api.listener.builder,\
```

Comment:
This is for testing (in the test folder). It should be removed when the http extension properly exposes it api.